client/src/components/ManageAccount/ManageAccountTables.js

```diff
@@ -185,12 +187,14 @@ export const Licenses = () => {
   );
   const data = useMemo(() => licenses, [licenses]);
   return (
-    <SectionTableWrapper
-      manualHeader={<SectionHeader isForList>Licenses</SectionHeader>}
-      manualContent
-    >
-      <TableTemplate attributes={{ columns, data }} />
-    </SectionTableWrapper>
+    !hideDataFiles && (
```

Comment:
Would these changes better live in `ManageAccount.js`?

Reply:
@rstijerina 👍🏼